src/main/java/com/stripe/net/ApiResourceTypeAdapterFactoryProvider.java

```diff
@@ -18,7 +18,6 @@
     factories.add(new BalanceTransactionSourceTypeAdapterFactory());
     factories.add(new ExternalAccountTypeAdapterFactory());
     factories.add(new PaymentSourceTypeAdapterFactory());
-    factories.add(new ReflectionCheckingTypeAdapterFactory());
   }
```

Comment:
What's this?

Reply:
This came in on https://github.com/stripe/stripe-java/pull/1197, I think I need to rebase this PR off a different master.